### PR TITLE
Fixed styling

### PR DESCRIPTION
**Pull request by AdrieanKhisbe**
few tweaks. fixing build: cf commit names

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "seneca"
+  "extends": "seneca",
+  "rules": {
```
Comment (mihaidma) [inline .eslintrc]: disabling rules not accepted :)

Comment (AdrieanKhisbe) [inline .eslintrc]: ahah.
Problem is that `lab` gives error for warning.
And this specific one should be a bit flexible.

So I found it more wise to disable the error, than to fix the ~30 capitalized name.

Comment (AdrieanKhisbe) [inline .eslintrc]: Doing so, I would have replace the snake case by pascal one.

Because I have to say, `Auth_token` and *so*, feels a bit bit bit weird

Comment (mihaidma) [inline .eslintrc]: ( authToken? )

Disabling still not accepted :)
We try to make uniform code across modules, disabling rules would cancel the purpose of having common rules. Let me know if you need me to help.

Comment (AdrieanKhisbe) [inline .eslintrc]: Pascal/camel here it is :)

**Comment by AdrieanKhisbe** [comment]
@mihaidma Et voilà!

**Comment by mihaidma** [comment]
Brilliant! Thank you very much for your effort.

**Comment by AdrieanKhisbe** [comment]
you're welcome :)

**Pull request merged**